client/lib/support/support-user/support-user-interop.js

```diff
@@ -82,7 +89,9 @@ export const rebootWithToken = ( user, token ) => {
 	debug( 'Rebooting Calypso with support user' );
 
 	store.set( STORAGE_KEY, { user, token } );
-	window.location.reload();
+
+	// This triggers a reboot
+	window.location.search = 'support_user=true';
```

Comment:
Not particularly, as we'll be switching to another user anyway so any search context won't be relevant. Potentially we might even want to default to sending the user to `/me`.